src/validation/__tests__/KnownDirectives-test.js

```diff
@@ -141,11 +145,26 @@ describe('Validate: Known directives', () => {
     );
   });
 
+  it('with well placed variable definition directive', () => {
```

Comment:
@mjmahone Please mark all tests as `Experimental` simular to this:
https://github.com/graphql/graphql-js/blob/1d7efa98e6190c5221106f8261c9bc5adeba97ca/src/language/__tests__/parser-test.js#L376
Also please add this prefix to tests from #1437

Reply:
Merged before I saw your comments: will fix up now.